lkp-exec/install

```diff
@@ -225,7 +225,7 @@ install_packages() {
 			install_ocfs2_tools "$ocfs2_tools_name"
 		fi
 	fi
-	
+
```

Comment:
similarly, pls merge with previous commit

Reply:
merged the commit.